scripts/hid_configurator/README.rst

```diff
@@ -24,6 +24,12 @@ Overview
 
 The script looks for nRF Desktop devices connected to the host through USB, Bluetooth®, or nRF Desktop dongle.
 The devices are identified based on Vendor ID.
+If you want to connect to your device which has different Vendor or Product ID you can run script with
```

Comment:
Move this under: `Debian/Ubuntu/Linux Mint` and merge it with the current description. This only affects Linux setup actually.